wallet/res/xml/preference_about.xml

```diff
@@ -46,10 +46,4 @@
         </Preference>
     </PreferenceCategory>
 
-	<Preference
-		android:key="firebase_instance_id_label"
-
-		android:persistent="false"
-		android:title="FIAM Instance ID" />
-
```

Comment:
`FirebaseInstanceId.getInstance().getId()` requires additional dependency - `'com.google.firebase:firebase-messaging:20.2.4'`. Additionally we do not use this version of `AboutFragment.java` anymore.